dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPIImpl.java

```diff
@@ -705,171 +712,171 @@ public Folder createFolders(String path, Host host, User user, boolean respectFr
+        if (!permissionAPI.doesUserHavePermission(folderToMove, PermissionAPI.PERMISSION_READ, user, respectFrontEndPermissions)) {
+            throw new DotSecurityException("User " + user.getUserId() != null?user.getUserId():"" + " does not have permission to read Folder " + folderToMove.getPath());
+        }
+
+        if (!permissionAPI.doesUserHavePermission(newParentHost, PermissionAPI.PERMISSION_CAN_ADD_CHILDREN, user, respectFrontEndPermissions)) {
+            throw new DotSecurityException("User " + user.getUserId() != null?user.getUserId():"" + " does not have permission to add to Folder " + newParentHost.getHostname());
+        }
+        boolean move = folderFactory.move(folderToMove, newParentHost);
+
+        this.systemEventsAPI.pushAsync(SystemEventType.MOVE_FOLDER, new Payload(folderToMove, Visibility.EXCLUDE_OWNER,
+                new ExcludeOwnerVerifierBean(user.getUserId(), PermissionAPI.PERMISSION_READ, Visibility.PERMISSION)));
+
+        return move;
+    }
+
+    @CloseDBIfOpened
+    public List<Folder> findSubFolders(Host host, boolean showOnMenu) throws DotHibernateException{
+        return folderFactory.findSubFolders(host, showOnMenu);
+    }
+
+    @CloseDBIfOpened
+    public List<Folder> findSubFolders(Folder folder,boolean showOnMenu)throws DotStateException, DotDataException {
+        return folderFactory.findSubFolders(folder, showOnMenu);
+    }
+
+    @CloseDBIfOpened
+    public List<String> getEntriesTree(Folder mainFolder, String openNodes,String view, String content, String structureInode,User user)
+            throws DotStateException, DotDataException, DotSecurityException {
+        Locale locale = user.getLocale();
+        TimeZone timeZone = user.getTimeZone();
+        Role[] roles = (Role[])APILocator.getRoleAPI().loadRolesForUser(user.getUserId()).toArray(new Role[0]);
+        boolean isAdminUser = APILocator.getUserAPI().isCMSAdmin(user);
+        return folderFactory.getEntriesTree(mainFolder, openNodes, view, content, structureInode, locale, timeZone, roles, isAdminUser, user);
+    }
+
+
+    @CloseDBIfOpened
+    public List<String> getFolderTree(String openNodes, String view,
+            String content, String structureInode,User user)
+            throws DotStateException, DotDataException, DotSecurityException {
+        Locale locale = user.getLocale();
+        TimeZone timeZone = user.getTimeZone();
+        Role[] roles = (Role[])APILocator.getRoleAPI().loadRolesForUser(user.getUserId()).toArray(new Role[0]);
+        boolean isAdminUser = APILocator.getUserAPI().isCMSAdmin(user);
+        return folderFactory.getFolderTree(openNodes, view, content, structureInode, locale, timeZone, roles, isAdminUser, user);
+    }
+
+    @CloseDBIfOpened
+    public List<Contentlet> getLiveContent(Folder parent, User user,boolean respectFrontEndPermissions) throws DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 151 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)